metron-platform/metron-parsing/metron-parsers/pom.xml

```diff
@@ -30,6 +30,15 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
   </properties>
   <dependencies>
+
+    <!-- Test dependencies needed preferentially -->
+    <dependency>
+      <groupId>com.github.fge</groupId>
+      <artifactId>jackson-coreutils</artifactId>
```

Comment:
Fixed issue with metron-parsers unit tests: `NoClassDefFoundError: Could not initialize class com.github.fge.jackson.JsonLoader`